grpc-compiler-plugin-tests/src/test/resources/test-src/package_11/grpc_service.bal

```diff
@@ -0,0 +1,68 @@
+// This is the server implementation of the unary blocking/unblocking scenario.
```

Comment:
add license header

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-grpc/pull/192/commits/a3ea94e1cbe73cb9d935070ad3b3bd9a137a041d